### PR TITLE
[ Application ] Fix Logistic Regression Example to work

### PR DESCRIPTION
**Pull request by jijoongmoon**
Current implementation of Logistic example is not working.
This PR inclues fixes.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: Applications/LogisticRegression/jni/main.cpp [hunk 1]
```diff
@@ -54,24 +67,110 @@ float stepFunction(float x) {
...
+  }
+
+  for (unsigned int i = train_count; i < train_count + batch_size; ++i) {
+
+    std::vector<float> o;
+    std::vector<float> l;
+    o.resize(feature_size);
+    l.resize(1);
+
+    if (!getData(dataFile, o, l, i)) {
```
Comment (zhoonit) [inline Applications/LogisticRegression/jni/main.cpp:130]: Could you check if this function always returns true?

File: Applications/LogisticRegression/jni/main.cpp [hunk 2]
```diff
@@ -54,24 +67,110 @@ float stepFunction(float x) {
...
+ * @param[in]  arg 2 : configuration file path
+ * @param[in]  arg 3 : resource path (dataset.txt or testset.txt)
  */
 int main(int argc, char *argv[]) {
-  if (argc < 3) {
-    std::cout << "./LogisticRegression Config.ini data.txt\n";
+  if (argc < 4) {
+    std::cout
+      << "./LogisticRegression train (| inference) Config.ini data.txt\n";
     exit(0);
```
Comment (zhoonit) [inline Applications/LogisticRegression/jni/main.cpp:158]:
```suggestion
    exit(1);
```

**Review by zhoonit — COMMENTED** [review]
Could you change

```diff
- test('app_logistic', e, args: [join_paths(res_path, 'LogisticRegression.ini'), join_paths(res_path, 'dataset1.txt')])
+ test('app_logistic', e, args: ['train', join_paths(res_path, 'LogisticRegression.ini'), join_paths(res_path, 'dataset1.txt')])
```

File: Applications/LogisticRegression/jni/main.cpp (hunk 1 above)
Comment (jijoongmoon) [reply Applications/LogisticRegression/jni/main.cpp:133]: fixed. Thanks.

File: Applications/LogisticRegression/jni/main.cpp (hunk 2 above)
Comment (jijoongmoon) [reply Applications/LogisticRegression/jni/main.cpp]: fixed. Thanks.

**Review by zhoonit — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**